packages/kbn-securitysolution-io-ts-alerting-types/src/actions/index.ts

```diff
@@ -25,27 +25,44 @@ export const RuleActionTypeId = t.string;
 export type RuleActionParams = t.TypeOf<typeof RuleActionParams>;
 export const RuleActionParams = saved_object_attributes;
 
+export type RuleActionFrequency = t.TypeOf<typeof RuleActionFrequency>;
+export const RuleActionFrequency = t.type({
+  summary: t.boolean,
+  notifyWhen: t.union([
```

Comment:
`frequency.notifyWhen` is always camelCased, even in API requests. When building the initial backend for this feature we ran into some issues with the functions that convert snake_casing to camelCasing in API requests when there's a nested parameter like the frequency, so I've replicated this schema in the security solution as well for consistency.